apps/docs/src/guide/wallets/access.md

```diff
@@ -27,6 +27,12 @@ provides greater access via its private key.
 
 <<< @/../../../packages/fuel-gauge/src/doc-examples.test.ts#wallets{ts:line-numbers}
 
+## Optional Provider
+
+You can choose not to pass through a provider argument on `Wallet` construction:
+
+<<< @/../../../packages/fuel-gauge/src/doc-examples.test.ts#wallet-optional-provider{ts:line-numbers}
```

Comment:
@FuelLabs/sdk-ts Just a reminder that code snippets should ideally come from the `apps/docs-snippets.`